.github/workflows/scripts/code_coverage_disable_check.py

```
@@ -0,0 +1,161 @@
+    except FileNotFoundError:
+        print(f"File not found: {file_path}")
+        return False
+    except PermissionError:
+        print(f"Permission denied: {file_path}")
+        return False
+    except (IOError, OSError) as e:
+        print(f"Error reading file {file_path}: {e}")
+        return False
+
+
+def check_code_coverage(files_or_dirs):
```

Ruff (0.8.2)

`check_code_coverage` is too complex (11 > 10)

(C901)